src/main/java/pages/yandex/HomePage.java

```diff
@@ -14,7 +14,6 @@
     public HomePage(WebDriver driver) {
```

Comment:
I don't want return to question that this pull request is incorrect. I can't put comment directly to line 12, that's why I add it here: add access modifier.
P.S. Future comments also can have info for which line I've put it.

Reply:
Still not updated.